.travis.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       env: WEBPACK_VERSION=latest JOB_PART=test
       script: npm run travis:$JOB_PART
     - <<: *test-latest
-      node_js: 4.3
+      node_js: 4.7
```

Comment:
`webpack-defaults@latest` (`v1.6.0`) would bump this aswell `npm run webpack-defaults`

Reply:
And for reference, we are going with `4.8` across the organization